tests/test_readme_setup.py

```
@@ -0,0 +1,89 @@
+    monkeypatch.setenv("DISCORD_TOKEN", "test-token-discord")
+
+    # Use isolated data_dir (README uses /tmp/ghdcbot-state; we use tmp_path)
+    config_content = EXAMPLE_CONFIG_PATH.read_text()
+    config_content = config_content.replace(
+        'data_dir: "/tmp/ghdcbot-state"',
+        f'data_dir: "{tmp_path}"',
+    )
+    config_path = tmp_path / "ghdcbot-config.yaml"
+    config_path.write_text(config_content)
+
+    config = load_config(str(config_path))
```

Ruff (0.14.14)

[error] Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)